src/assets/css/_colors.scss

```diff
@@ -10,6 +10,8 @@ $info: #1cbdee;
 $light-grey: #cbcbcb;
 $light-red: #f5eeee;
 $yellow: #f9d342;
+$darkenBlue: #1e88e5;
+$greenAccent: #eeeeee;
 /* background anim colors */
```

Comment:
- Remove comment
- CSS uses `hyphen-case` not `camelCase`

Reply:
Changes applied